processing/data_collection/gazette/spiders/base.py

```diff
@@ -18,6 +20,63 @@ def __init__(self, start_date=None, *args, **kwargs):
                 self.start_date = parsed_data.date()
 
 
+class DiarioMunicipalSigpubGazetteSpider(BaseGazetteSpider):
+    custom_settings = {"USER_AGENT": UserAgent().random}
+
+    def start_requests(self):
+        yield scrapy.Request(self.CALENDAR_URL, callback=self.parse_calendar)
+
+    def parse_calendar(self, response):
+        formdata = {
+            "calendar[_token]": response.xpath(
+                "//input[@id='calendar__token']/@value"
+            ).get()
+        }
+        available_dates = rrule(
+            freq=DAILY, dtstart=self.FIRST_AVAILABLE_DATE, until=date.today()
+        )
+        for query_date in available_dates:
+            formdata.update(
+                {
+                    "calendar[day]": str(query_date.day),
+                    "calendar[month]": str(query_date.month),
+                    "calendar[year]": str(query_date.year),
+                }
+            )
+            editions = {
+                "regular": self.GAZETTE_INFO_URL,
+                "extra": self.EXTRA_GAZETTE_INFO_URL,
+            }
+            for edition_type, url in editions.items():
+                yield scrapy.FormRequest(
+                    url=url,
+                    formdata=formdata,
+                    meta={"date": query_date, "edition_type": edition_type},
+                    callback=self.parse_gazette_info,
+                )
```

Comment:
```suggestion
        yield scrapy.FormRequest(
                    url= self.GAZETTE_INFO_URL,
                    formdata=formdata,
                    meta={"date": query_date, "edition_type": "regular"},
                    callback=self.parse_gazette_info,
        )
        yield scrapy.FormRequest(
                    url=self.EXTRA_GAZETTE_INFO_URL,
                    formdata=formdata,
                    meta={"date": query_date, "edition_type": "extra"},
                    callback=self.parse_gazette_info,
        )
```

Less indentations and loop control structures. ;)

Reply:
Done :)